### DISCUSSION TITLE
github-profile-trophy

### DISCUSSION THREAD

**Discussion by NeonGamerBot-QK** [catppuccin/catppuccin#2486] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!
